src/braket/aws/aws_quantum_job.py

```diff
@@ -249,6 +251,7 @@ def __init__(self, arn: str, aws_session: AwsSession | None = None, quiet: bool
         else:
             self._aws_session = AwsQuantumJob._default_session_for_job_arn(arn)
         self._metadata = {}
+        self._tasks = None
```

Comment:
In case people's workflows iterate through this list with or without tasks
```suggestion
        self._tasks = []
```